modules/client-configuration/pages/supported-features-amazon.adoc

```diff
@@ -0,0 +1,160 @@
+[[supported-features-amazon]]
+= Supported {amazonlinux} Features
+
+
+This table lists the availability of various features on {amazonlinux} clients.
+
+[IMPORTANT]
+====
+The operating system you run on a client is supported by the organization that supplies the operating system.
+{amazonlinux} is supported by Amazon.
+====
+
+The icons in this table indicate:
+
+* {check} the feature is available on both Salt and traditional clients
+* {cross} the feature is not available
+* {question} the feature is under consideration, and may or may not be made available at a later date
+* {trad} the feature is supported only on traditional clients
+* {salt} the feature is supported only on Salt clients
+
+
+[cols="1,1", options="header"]
+.Supported Features on {amazonlinux} Operating Systems
+|===
+
+| Feature
+| {amazonlinux}{nbsp}2
+
+| Client
+| {salt}
+
+| Operating system packages
+| {salt}
+
+| Registration
+| {salt}
+
+| Install packages
+| {salt}
+
+| Apply patches (requires CVE ID)
+| {question}
+
+| Remote commands
+| {salt}
+
+| System package states
+| {salt}
+
+| System custom states
+| {salt}
+
+| Group custom states
+| {salt}
+
+| Organization custom states
+| {salt}
+
+| System set manager (SSM)
+| {salt}
+
+| Service pack migration
+| N/A
+
+| Basic Virtual Guest Management {star}
+| {salt}
+
+| Advanced Virtual Guest Management {star}
+| {salt}
+
+| Virtual Guest Installation (Kickstart), as Host OS
+| {cross}
+
+| Virtual Guest Installation (image template), as Host OS
```

Comment:
What does "as Host OS" mean here? Doesnt' "virtual guest installation" contradict "host OS"?

Reply:
TBH, the matrix is copied from ORacle, so no idea. @jcayouette @Loquacity @keichwa

Reply:
@paususe seems you told the doc squad to add this (at confluence), so ... what does it mean?